netctl/netctl.go

```diff
@@ -401,3 +402,14 @@ func dumpList(ctx *cli.Context, list []map[string]interface{}) {
 	os.Stdout.Write(content)
 	os.Stdout.WriteString("\n")
 }
+
+func showVersion(ctx *cli.Context) {
```

Comment:
Please call it `String()`. Go automatically uses this method when handling `%v` and all sorts of stuff.

Other than that, I am not opposed to moving forward with this patch as-is. Handling API version would require a lot more code, and I think is out of scope at this point.